src/containers/FormWrapper/FormWrapper.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import './FormWrapper.css';
+import HeaderForm from '../../components/Forms/HeaderForm/HeaderForm';
+import TodosForm from '../../components/Forms/TodosForm/TodosForm'
+import FooterForm from '../../components/Forms/FooterForm/FooterForm';
+
+import {connect} from 'react-redux'
+
+import * as actionCreators from '../../store/actions/manipulateTodoState';
+
+import {Route} from 'react-router-dom';
+
+const FormWrapper = (props) => {
+
+
+    return (
+        <div className="form-container">
+
+            {/* #############      HEADER     ############ */}
+            <Route path="/"
+                   render={() => (
+                       <HeaderForm
+                           postTodoHandler={props.postTodoHandler}
+                       />
+                   )}
+            />
+
+
+            <Route path="/" exact
+                   render={() => (
+                       <TodosForm
+                           todos={props.todos}
+                           deleteTodoHandler={props.deleteTodoHandler}
+                           putTodoHandler={props.putTodoHandler}
+
+                       />)
+                   }
+            />
+
+
+            <Route path="/completed-todos" exact
+                   render={() => (
+                       <TodosForm
+                           todos={props.completedTodos}
+                           deleteTodoHandler={props.deleteTodoHandler}
+                           putTodoHandler={props.putTodoHandler}
+                       />
+                   )}
+            />
+
+
+            <Route path="/not-completed-todos" exact
+                   render={() => (
+                       <TodosForm
+                           todos={props.notCompletedTodos}
+                           deleteTodoHandler={props.deleteTodoHandler}
+                           putTodoHandler={props.putTodoHandler}
+                       />
+
+                   )}
+            />
+
+            {/* #############      FOOTER     ############ */}
+            <Route path="/" component={FooterForm}/>
+
+        </div>
+    );
+};
+
+
```

Comment:
There is also a technique like this:

```js
const mapStateToProps = state => {
  return {
     todos: state.todos,
     completedTodos: state.todos.filter(...),
     notCompletedTodos: state.todos.filter(...),
  }
}
```

Reply:
I did not know that but it looks clearer ! Thank you!